geomesa-tools/conf/sfts/adsbx/reference.conf

```diff
@@ -0,0 +1,248 @@
+caches = {
+  spdtyp_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Ground Speed" },
+      "1" = { code = "Ground Speed Reversing" },
+      "2" = { code = "Indicated Air Speed" },
+      "3" = { code = "True Air Speed" }
+    }
+  }
+  wtc_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Light" },
+      "2" = { code = "Medium" },
+      "3" = { code = "Heavy" }
+    }
+  }
+  species_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Land Plane" },
+      "2" = { code = "Sea Plane" },
+      "3" = { code = "Amphibian" },
+      "4" = { code = "Helicopter" },
+      "5" = { code = "Gyrocopter" },
+      "6" = { code = "Tiltwing" },
+      "7" = { code = "Ground Vehicle" },
+      "8" = { code = "Tower" }
+    }
+  }
+  engtype_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "None" },
+      "1" = { code = "Piston" },
+      "2" = { code = "Turboprop" },
+      "3" = { code = "Jet" },
+      "4" = { code = "Electric" }
+    }
+  }
+  engmount_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Unknown" },
+      "1" = { code = "Aft Mounted" },
+      "2" = { code = "Wing Buried" },
+      "3" = { code = "Fuselage Buried" },
+      "4" = { code = "Nose Mounted" },
+      "5" = { code = "Wing Mounted" }
+    }
+  }
+  trt_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Unknown" },
+      "1" = { code = "Mode-S" },
+      "2" = { code = "ADS-B (unknown version)" },
+      "3" = { code = "ADS-B 0 – DO-260" },
+      "4" = { code = "ADS-B 1 – DO-260 (A)" },
+      "5" = { code = "ADS-B 2 – DO-260 (B)" }
+    }
+  }
+  tt_codes = {
+    type = "simple"
+    data = {
+      "a" = { code = "Altitude" },
+      "s" = { code = "Speed" }
+    }
+  }
+  vsit_codes = {
+    type = "simple"
+    data = {
+      "0" = { code = "Barometric" },
+      "1" = { code = "Geometric" }
+    }
+  }
+  adsbx_static = {
```

Comment:
this cache would need to be populated with something, right? i'm hesitant to include it in the default, given that it won't really work out of the box (or it will just throw a lot of errors/timeouts?). maybe we should have separate converters that include the enrichment, e.g. `adsbx-enriched`, `adsbx-multi-enriched`, etc. Per Austin's comments, some more mention of this in the docs would be helpful too.

Reply:
Can caches be defined within a converter?

Reply:
Or are you talking about making references to a cache within a different converter?

Reply:
yeah, have one set of converters that don't invoke the redis cache, and then a second set that do (but require some additional work, since you'd have to set up redis and populate it)

Reply:
i think caches have to be defined outside converters